src/llvm-multiversioning.cpp

```diff
@@ -542,7 +542,7 @@ void CloneCtx::check_partial(Group &grp, Target &tgt)
         grp.clone_fs.insert(i);
         all_origs.insert(orig_f);
     }
-    std::set<Function*> sets[2]{all_origs, {}};
+    std::set<Function*> sets[2]{all_origs, std::set<Function*>{}};
```

Comment:
Does leaving it out work?

Reply:
not sure, I'm just sort of doing a random walk to figure out what will make the buildbot happy